docs/src/main/sphinx/release/release-360.md

```diff
@@ -0,0 +1,71 @@
+# Release 360 (XXX Jul 2021)
+
+## General
+
+* Reduce number of requests from coordinator to workers ({issue}`8460`)
+* Improve query performance by sending collected dynamic filters from coordinator to workers. ({issue}`5183`)
+* Improve performance of inequality joins where join condition sides have different type. ({issue}`8500`)
+* Fix query failure when there is a constant predicate on some
+  column `col` (e.g `col=1`), followed by  `ORDER BY col` and `LIMIT`. ({issue}`8535`)
+* Fix SHOW CREATE SCHEMA. ({issue}`8598`)
+* Fix query failure when running `SHOW CREATE SCHEMA information_schema`. ({issue}`8600`)
+* Improve performance of queries with `WHERE` predicates. ({issue}`8624`)
+* Disallow `DROP SCHEMA` if the schema is not empty. ({issue}`8660`)
```

Comment:
This was actually a pretty serious data loss scenario.

Fortunately, it was affecting only few connectors (MySQL, MemSQL, ClickHouse, ??) (plus, any custom connectors people could have).

How do we reflect this in the release notes?

Reply:
It's a serious data loss only if the expectation is that DROP SCHEMA will fail if it contains tables. I don't think it's unexpected that DROP SCHEMA *might* delete data, though.

Reply:
That's "serious data loss" if you think about `DROP SCHEMA RESTRICT`.
it's not documented, but we treat `DROP SCHEMA RESTRICT` as same as `DROP SCHEMA`. Users executing `DROP SCHEMA RESTRICT` would definitely not expect their tables to vanish.